src/index.html

```diff
@@ -20,26 +20,32 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-i18next/1.2.0/jquery-i18next.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
 
+  <script src="./js/bootstrap-wysiwyg.js"></script>
+
 <script type="text/javascript">
 
-// 現在表示中の機能ID
+// Function ID currently displayed
```

Comment:
Please move all JavaScript into JS file and make sure it will work when loaded asynchronously.

Reply:
resolved : pull request #10